rs/execution_environment/src/canister_manager/tests.rs

```diff
@@ -7865,3 +7892,58 @@ fn chunk_store_counts_against_subnet_memory_in_initial_round_computation() {
         .unwrap_err();
     assert_eq!(error.code(), ErrorCode::SubnetOversubscribed);
 }
+
+fn run_canister_in_wasm_mode(is_wasm64_mode: bool, execute_ingress: bool) -> (Cycles, Cycles) {
```

Comment:
Maybe leave a small doc comment that explains what this helper is doing?

Reply:
done, thanks.